src/Sql/Sql/OutboundFirewallRules/Cmdlet/GetAzureSqlServerOutboundFirewallRule.cs

```diff
@@ -0,0 +1,99 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.OutboundFirewallRules.Model;
+using Microsoft.Rest.Azure;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.OutboundFirewallRules.Cmdlet
+{
+    /// <summary>
+    /// Defines the Get-AzSqlServerOutboundFirewallRules cmdlet
+    /// </summary>
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlServerOutboundFirewallRule", SupportsShouldProcess = true, ConfirmImpact = ConfirmImpact.None)]
+    [OutputType(typeof(AzureSqlServerOutboundFirewallRulesModel))]
+    public class GetAzureSqlServerOutboundFirewallRule : AzureSqlServerOutboundFirewallRulesCmdletBase
+    {
+        /// <summary>
+        /// Gets or sets the allowed FQDN of the Azure Sql Database Server Outbound Firewall Rule
+        /// </summary>
+        [Parameter(Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            Position = 2,
+            HelpMessage = "The Azure Sql Database Server Outbound Firewall Rule FQDN.")]
+        [Alias("Name")]
+        [ValidateNotNullOrEmpty]
+        [SupportsWildcards]
+        public string AllowedFQDN { get; set; }
```

Comment:
I suggest to move the definition of AllowedFQDN to AzureSqlServerOutboundFirewallRulesCmdletBase

Reply:
The AllowedFQDN parameter is optional in `Get-AzSqlServerOutboundFirewallRules`. Therefore, Mandatory = false for Get but it is true for New and Remove.

Reply:
Make sense.